### DISCUSSION TITLE
Error Install Docker Compose lastet Image

### DISCUSSION THREAD

**Discussion by haydarle** [outline/outline#6655] [Self Hosting]
Hello,

i will try to install outline with docker-compose on portainer.io. I have created a Docker-compose file and a *.env File.
After Deploy, i become following Notification on the outline-Container.
I dont know what is the container say to me. The other Containers (postgres, redis and https-portal are running.

> `yarn run v1.22.19
> $ node ./build/server/index.js
> /opt/outline/node_modules/sequelize/lib/sequelize.js:58
>       options.dialect = urlParts.protocol.replace(/:$/, "");
>                                           ^
> TypeError: Cannot read properties of null (reading 'replace')
>     at new Sequelize (/opt/outline/node_modules/sequelize/lib/sequelize.js:58:43)
>     at new Sequelize (/opt/outline/node_modules/sequelize-typescript/dist/sequelize/sequelize/sequelize.js:16:9)
>     at createDatabaseInstance (/opt/outline/build/server/storage/database.js:25:10)
>     at Object.<anonymous> (/opt/outline/build/server/storage/database.js:95:39)
>     at Module._compile (node:internal/modules/cjs/loader:1376:14)
>     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
>     at Module.load (node:internal/modules/cjs/loader:1207:32)
>     at Module._load (node:internal/modules/cjs/loader:1023:12)
>     at Module.require (node:internal/modules/cjs/loader:1235:19)
>     at Hook.Module.require (/opt/outline/node_modules/dd-trace/packages/dd-trace/src/ritm.js:85:33)
>     at require (node:internal/modules/helpers:176:18)
>     at Object.<anonymous> (/opt/outline/build/server/collaboration/PersistenceExtension.js:12:17)
>     at Module._compile (node:internal/modules/cjs/loader:1376:14)
>     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
>     at Module.load (node:internal/modules/cjs/loader:1207:32)
>     at Module._load (node:internal/modules/cjs/loader:1023:12)
> Node.js v20.11.1
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

**This is my Docker-Compose File and my *.env File**

version: "3.2"
services:

  outline:
    image: docker.getoutline.com/outlinewiki/outline:0.75.1
    ports:
      - "3000:3000"
    volumes:
      - storage-data:/var/lib/outline/data
    depends_on:
      - postgres
      - redis

  redis:
    image: redis
    ports:
      - "6379:6379"
    volumes:
      - ./redis.conf:/redis.conf
    command: ["redis-server", "/redis.conf"]
    healthcheck:
      test: ["CMD", "redis-cli", "ping"]
      interval: 10s
      timeout: 30s
      retries: 3

  postgres:
    image: postgres
    ports:
      - "5432:5432"
    volumes:
      - database-data:/var/lib/postgresql/data
    healthcheck:
      test: ["CMD", "pg_isready"]
      interval: 30s
      timeout: 20s
      retries: 3
    environment:
      POSTGRES_USER: 'user'
      POSTGRES_PASSWORD: 'pass'
      POSTGRES_DB: 'outline'

  https-portal:
    image: steveltn/https-portal
    ports:
      - '80:80'
      - '443:443'
    links:
      - outline
    restart: always
    volumes:
      - https-portal-data:/var/lib/https-portal
    healthcheck:
      test: ["CMD", "service", "nginx", "status"]
      interval: 30s
      timeout: 20s
      retries: 3
    environment:
      DOMAINS: 'docs.mycompany.com -> http://outline:3000'
      STAGE: 'production'
      WEBSOCKET: 'true'
      CLIENT_MAX_BODY_SIZE: '0'

volumes:
  https-portal-data:
  storage-data:
  database-data:

**Env File:**

The and File is the standard sample file.

**Comment by Its4Nik** [comment]
Afaik you cant use the standart .env file. 

I would recommend an installer, like [EasyOutline](https://github.com/its4nik/easyoutline), yeah i know i've written it myself but as far as my testing goes it works without a problem on ubuntu server, i am also working on a v2 which will add more functionality and a better documentation. I would really apreciate some feedback :)

**Comment by Frixs** [comment]
I have the same issue when trying to install.
https://github.com/outline/outline/discussions/8224
